src/components/ButtonDelete/ButtonDelete.scss

```diff
@@ -6,13 +6,13 @@
 .delete-button {
     all: unset;
     background-color: $color-red;
-    color: #FFFFFF;
     width: 100%;
     height: 36px;
     font-weight: 600;
     border-radius: 20px;
     text-align: center;
     @include body-small;
+    color:#ffff;
```

Comment:
You cannot put this line between @include body-small and @include tablet as it will create an error of nesting style.
Please just rename this color at its original position.

Reply:
Got it, readjusted to previous state. I added !important just to match the color